documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
+
+Additional capital contributions can be voted on and approved by the
+Corporate Overlords during a plotting session.
+
+Question for Howard: how do we allow members *or* CO's to contribute
+funds directly to specific BIFFUD projects without having equity impact?
```

Comment:
If they're for a specific BIFFUD project, I think that's somewhat easier: to us, they'd be categorized as a grant or (non-tax-deductible) donation, as income directly into the budget of the BIFFUD project (below). And not as BIFFUD corporate profit or any general operating budget of the company. Grantee makes no claim of ownership.

But yes, check _everything_ with the lawyer.